### ISSUE TITLE
Highlights cover pictures are not the actual Highlights cover pictures!

### ISSUE THREAD

**Issue by UweLau** [instaloader/instaloader#844]
**Describe the bug**
When downloading the highlights the downloaded cover pictures do not match the ones that are showing in the web interface.

**To Reproduce**
instaloader PROFILE --highlights (--login USER --password PASS)

**Expected behavior**
I want the real cover pictures and not some size-down and cut versions of the first picture in the highlight. In the web interface I see only a 150x150 version. I do not know if there are bigger versions of these cover pictures.

**Instaloader version**
4.5.3

**Comment by aandergr** [comment]
Is this the same problem that has been reported in #668? Can you give an example to reproduce this?

**Comment by UweLau** [comment]
I found this user where we have two types of highlight cover pictures: `simone_f_inked`
This is the command I am using:

`instaloader --stories --highlights --igtv --fast-update --login=xxx --password=XXXXX simone_f_inked`

I do not follow the account but it seems that does not matter as I have this behavior on private accounts as well.

Type 1: No special picture is used
See the "Shootings" highlight. Instagram uses a part of the first picture to generate the picture. The pictures do match.
Type 2: Some special uploaded picture is used
See the "🙋🏼‍♀️🌞🤳" highlight. You can see the picture of her in the white bikini on the beach on the web interface but Instaloader downloads (or generates?) a different picture. It may be the case the cropped down version of the first picture always exists somewhere in the system but is then not used and instead the uploaded picture is used?

**Comment by MrJmpl3** [comment]
I think this issue need be reopen because the problem persists

**Comment by MrJmpl3** [comment]
> > **Edit**
> > @aandergr Can add a param to download "cover_media_cropped_thumbnail" and "cover_media" together?
> 
> Is that of any use?

Yes, is to download both, if the problem persist, download both make a temporal solution.

I know cover_media_cropped_thumbnail is a low resolution, but sometime is a different to cover_media

**Comment by aandergr** [comment]
> Yes, is to download both, if the problem persist, download both make a temporal solution.

I'd favor fixing the problem, rather than implementing to download both. The downloaded cover picture should match the ones that are shown in the web interface.

**Comment by MrJmpl3** [comment]
> 
> 
> > Yes, is to download both, if the problem persist, download both make a temporal solution.
> 
> I'd favor fixing the problem, rather than implementing to download both. The downloaded cover picture should match the ones that are shown in the web interface.

But, If you make that, We cannot download the other cover photo, The idea of a scraper is get the most data possible

**Comment by aandergr** [comment]
> But, If you make that, We cannot download the other cover photo

The issue descriptions says it is some size-down and cut versions of the first picture in the highlight, so you still get it.

**Comment by MrJmpl3** [comment]
> 
> 
> > But, If you make that, We cannot download the other cover photo
> 
> The issue descriptions says it is some size-down and cut versions of the first picture in the highlight, so you still get it.

Not really, the images are differents, the issue is to check if can download a HQ image of the other cover of the highlights.

**Comment by UnixCro** [comment]
At the moment it is only possible to download the Highlight Cover with the JavaScript console with a computer with a poor resolution of 150x150.  If you should manage to get a higher resolution it would be just awesome.  I've tried so many websites and so many, but I haven't managed to get higher than 150.  Instaloader is currently going differently as far as I know, it looks at the cover to see if it is already in the highlights and then scales it accordingly.

**Comment by MrJmpl3** [comment]
@aandergr Please, I think is only add some lines to download the attribute 'cover_cropped_url'.

I can make the PR, but I am newbie in Python.

**Comment by aandergr** [comment]
@MrJmpl3 I agree that the bug described in this issue is likely easy to fix, hence I added the label 'good first issue'. Feel free to create a PR.

**Comment by UnixCro** [comment]
Just tested again. `instaloader` does not download the highlight cover, but takes the first frame from the highlight and scales it to `640x640`
